<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSerialSettingsClass</class>
 <widget class="QWidget" name="DialogSerialSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>串口设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:#E0E0E0;
font-family:Microsoft YaHei;
background: #323232;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,20">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Serial Port Setting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">
QComboBox
{
	color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;
}
QGroupBox{border:1px solid #212121;}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Serial port Number</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cmbName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Serial port Rate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cmbBaudRate"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Check Bits</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cmbParity"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Data Bits</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cmbDataBits"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Stop Bits</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cmbStopBits"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Flow Control</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cmbFlowControl"/>
          </item>
          <item row="6" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="btnOpenSerial">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">
QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;

}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCloseSerial">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">
QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;

}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
              </property>
              <property name="text">
               <string>Close</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Device Debugging</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">QGroupBox{border:1px solid #212121;}
QPushButton
{
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
color:white;

}
QPushButton:hover
{
	background: #4A4A4A;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:pressed
{
	background: #787878;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}
QPushButton:disabled
{
	background: #ABABAB;
	border: 1px solid rgba(255, 255, 255, 0.2);
	border-radius: 5px;
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout" rowstretch="0,1,0,0,1">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Data Log</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnClearLog">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QTextEdit" name="textEditLog">
            <property name="styleSheet">
             <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Data Send</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnClearSend">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSend">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QTextEdit" name="textEditSend">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:white;
background: #262626;
border: 1px solid rgba(255, 255, 255, 0.2);
border-radius: 4px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
